specification/components/schemas/parameters/ListType.yaml

```diff
@@ -19,20 +19,43 @@ properties:
           type: array
           minItems: 1
           maxItems: 1
+          description: |
+            | Field                            | Cardinality | Notes                                  |
+            | -------------------------------- | ----------- | -------------------------------------- |
+            | `eRS-ReferralListSelector`       | `0..1`      |                                        |
+            | `eRS-CommunicationListSelector`  | `0..1`      |                                        | 
           items:
-              type: object
-              required:
-                - system
-                - code
-              properties:
-                system:
+            anyOf:
+              - title: eRS-ReferralListSelector
+                type: object
+                required:
+                 - system
+                 - code
+                properties:
+                  system:
                     type: string
                     enum:
                       - https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReferralListSelector-1
                     example: 'https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReferralListSelector-1'
-                code:
+                  code:
```

Comment:
Looking at https://fhir.nhs.uk/STU3/OperationDefinition/eRS-FetchWorklist-Operation-1 the listType parameter should be a CodeableConcept rather than a Coding which this structure seems to map to.

Reply:
Checked it with Jack, it seems to be correct, the codeableconcept definition is higher up.